src/GHC/RTS/Events/Binary.hs

```diff
@@ -813,6 +830,28 @@ heapProfParsers =
         ])
       (return ())
     return $! HeapProfCostCentre {..}
+  , VariableSizeParser EVENT_IPE $ do
+    payloadLen <- get :: Get Word16
+    itInfo <- get
+    itTableName <- getTextNul
+    itClosureDescText <- getTextNul
+    let itClosureDesc :: Int = read (T.unpack itClosureDescText)
```

Comment:
nit: To avoid `String` conversion

```suggestion
    itClosureDesc <- either fail (return . fst) $ TR.decimal itClosureDescText
```
assuming `Data.Text.Read` is imported as `TR`.